webtau-docs/znai/GraphQL/getting-started.md

```diff
@@ -0,0 +1,57 @@
+:include-markdown: installation-groovy-runner.md
+
+# Minimal Groovy Setup
+
+Generate webtau examples 
```

Comment:
A lot of this is almost identical to the equivalent section in the REST API docs.  Would be awesome if znai supported a paramaterised version of `include-markdown` so we could just replace the relevant file names and REST vs. GraphQL words.

Reply:
I see. We could in theory use templates that are already there.

Reply:
Which templates?  My point is this particular page is very similar for REST and GraphQL with the exception of the example names, file names and the odd remark here and there.  Having 10 individual markdowns for each section that we compose isn't that user friendly.